CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - [UIMARCAUTH-387](https://issues.folio.org/browse/UIMARCAUTH-387) Change auto-open a record functionality.
 - [UIMARCAUTH-405](https://issues.folio.org/browse/UIMARCAUTH-405) Display a generic error message during authority file creation if validation is missing.
+- [UIMARCAUTH-408](https://issues.folio.org/browse/UIMARCAUTH-408) Import `useUserTenantPermissions` from `@folio/stripes/core`.
```

Comment:
`5.0.1` has been released already. Please create a new section for `5.1.0` release